xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -58,7 +58,16 @@
 using namespace WINDOWING;
 using namespace std::chrono_literals;
 
-#define MAX_DISPLAYS 32
+namespace
+{
+constexpr int MAX_DISPLAYS = 32;
+constexpr NSString* DEFAULT_SCREEN_NAME = @"Default";
```

Comment:
Why `NSString*`?  They are used as `const char*` params in `GetInt()`.
Besides, if you need a representation as `NSString*` you can just box it: `@(myCharConst)`

Reply:
nice tip, will be adjusted